### ISSUE TITLE
As an admin user I want to be able to make invoices automatically

### ISSUE THREAD

**Issue by ivan-unfolds** [MyPitit/umhan#3]

**Comment by Rachelhrpiper** [comment]
Yes and in order to do this we need to be able to filter and group according to the category of 'University'

E.g. If less than 3 people = £90 each
If a third person joining 2 already with membership = £65
If 3 people =£245
If 3- 10 people = £245

**Comment by MyPitit** [comment]
Thanks Rachel
